Drop-In/src/main/res/layout/bt_fragment_supported_payment_methods.xml

```diff
@@ -0,0 +1,94 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/bt_base_background"
+    android:minHeight="200dp"
+    tools:context="com.braintreepayments.api.DropInActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:id="@+id/bt_vaulted_payment_methods_wrapper"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:visibility="gone">
+
+            <RelativeLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingTop="@dimen/bt_choose_existing_payment_label_padding_top">
+
+                <Button
+                    android:id="@+id/bt_vault_edit_button"
+                    style="@style/Widget.AppCompat.Button.Borderless.Colored"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentEnd="true"
+                    android:text="@string/bt_edit"
+                    android:theme="@style/bt_edit_button"
+                    android:visibility="invisible" />
+
+                <TextView
+                    android:id="@+id/bt_vaulted_payment_methods_header"
+                    style="@style/bt_choose_payment_label"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@id/bt_vault_edit_button"
+                    android:layout_alignBottom="@id/bt_vault_edit_button"
+                    android:layout_alignParentStart="true"
+                    android:layout_marginStart="20dp"
+                    android:gravity="center_vertical"
+                    android:text="@string/bt_recent" />
```

Comment:
In Jira ticket #DTBTSDK-66, design wanted us to change "Recent" to "Saved" in Drop In. We didn't include that change in our latest iOS Drop In major version, but we can plan to next time.

Reply:
Yeah i'm down to make this change. We'll get a few rounds of polish in before GA for sure.